src/ui/server/content-strings/pages/quote.ts

```diff
@@ -6,7 +6,7 @@ const BUYER_BODY = {
   DETAILS: {
     INTRO: 'What counts as a government or public sector body?',
     BODY_1: 'This means a formally established organisation that is, at least in part, publicly funded to deliver a public or government service.',
-    BODY_2: 'For example, a central government department, a local authority or a public utlity company.',
+    BODY_2: 'For example, a central government department, a local authority or a public utility company.',
```

Comment:
Seems this change is actually valid, it should be `utlity` instead of `utility`, this is in the design, it is:

"_a service (such as light, power, or water) provided by a public utility._"

<img width="450" alt="Screenshot 2023-09-25 at 15 34 44" src="https://github.com/UK-Export-Finance/exip/assets/2463648/adbd58ee-7d37-4864-af16-80000b054ed6">

Reply:
Confirmed by design, the change is correct